lib/chef/knife/bootstrap/windows-chef-client-msi.erb

```diff
@@ -191,9 +191,9 @@ echo Writing validation key...
 echo Validation key written.
 @echo on
 
-<% if @config[:encrypted_data_bag_secret] -%>
+<% if @config[:secret] -%>
 > <%= bootstrap_directory %>\encrypted_data_bag_secret (
- <%= encrypted_data_bag_secret %>
```

Comment:
Won't this break people who are using encrypted_data_bag_secret already? We should use `secret` instead of encrypted_data_bag_secret, and warn if `encrypted_data_bag_secret` is specified that they should use `secret` instead, so we can deprecate it later. We might even be able to use deprecation support built into Chef.